src/pybind/mgr/nfs/module.py

```diff
@@ -24,65 +25,65 @@ def __init__(self, *args, **kwargs):
 
     @CLICommand('nfs export create cephfs', perm='rw')
     def _cmd_nfs_export_create_cephfs(self, fsname: str, clusterid: str, binding: str,
-                                      readonly: bool=False, path: str='/'):
+                                      readonly: bool=False, path: str='/') -> Tuple[int, str, str]:
```

Comment:
could you take this opportunity to annotate all the functions in this source file and add an section in `src/mypy.ini` to ensure that this file is annotated in future?

Reply:
I will do it in a different PR. I have created a tracker ticket for it: https://tracker.ceph.com/issues/50816